bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/InternalHooks.java

```diff
@@ -1,13 +1,40 @@
 package com.bugsnag.android;
 
+import static com.bugsnag.android.Bugsnag.client;
+
 import kotlin.Unit;
 import kotlin.jvm.functions.Function0;
 
 class InternalHooks {
 
-    private InternalHooks() {}
+    private InternalHooks() {
+    }
 
     public static void setEventStoreEmptyCallback(Client client, Function0<Unit> callback) {
         client.eventStore.setOnEventStoreEmptyCallback(callback);
     }
+
+    public static void deliver(Client client, Event event) {
+        client.deliveryDelegate.deliver(event);
+    }
+
+
+    public static Event createEmptyANR(Long exitInfoTimeStamp) {
```

Comment:
nit: `exitInfoTimeStamp` should be a primitive

```suggestion
   static Event createEmptyANR(long exitInfoTimeStamp) {
```